### ISSUE TITLE
glitch in trees (Zelda Botw)

### ISSUE THREAD

**Issue by dml360** [yuzu-emu/yuzu#7942]
attached is a video of a glitch that occurs quite often in the game "Zelda BOTW"

hardware:

i5 8400
gtx 1060 6gb
16gb ram 2666mhz

emulator setings:

Vulkan
gpu high
2x 1440p

https://youtu.be/NNlOVZK2MLA

https://www.youtube.com/watch?v=tNNzuKOjNuY&ab_channel=DyogeMartins

**Comment by G-Spawn** [comment]
Please provide a log file so we know more.

**Comment by dml360** [comment]
> Forneça um arquivo de log para que possamos saber mais.

Where can I get the log?

**Comment by G-Spawn** [comment]
https://yuzu-emu.org/help/reference/log-files/

**Comment by dml360** [comment]
thanks, as soon as I go through this glitch again I will do this procedure.

Another point that I would like to mention is when we press Y to see the "path of the hero" after zooming in, the green line disappears and there are numerous green micro dots passing across the screen

I'll do the log procedure and send both (but I don't know if the hero's path glitcj will work)

**Comment by dml360** [comment]
> Forneça um arquivo de log para que possamos saber mais.

https://mega.nz/file/T5lyiLSS#Bgm217s85yLeVwkeLpR2CPB6FrYxYc_OCFRZD5ceneA

**Comment by G-Spawn** [comment]
Please try using docked mode in the emulator, auto cpu accuracy, disable async shaders in config graphics advanced, and try updating gpu drivers.

**Comment by dml360** [comment]
> Por favor, tente usar o modo encaixado no emulador, precisão automática da CPU, desative os shaders assíncronos nos gráficos de configuração avançados e tente atualizar os drivers gpu.

I did what you said and the problem continued,
something "curious" is that this only starts to happen after about 20~40 minutes of gameplay (according to my experiences) and the "hero's path" problem hasn't changed either

**Comment by G-Spawn** [comment]
Please try using vulkan when you can to compare. OGL still has room for improvement on yuzu for botw.

**Comment by quanghuy1242** [comment]
I have the same issue as him. Every single tree keeps flickering in some area https://streamable.com/54iz8d

**Comment by dennajort** [comment]
I do see the same problem. And it's only on trees that are at a certain distance. Once you go near they stop flickering, and when you move away they start flickering again.
Using Vulkan, async shaders disabled, auto cpu accuracy, docked mode, gpu drivers up to date

**Comment by SnoopyCodes** [comment]
I also see the same problem. I think it's related to how BOTW renders trees- there's a youtube short I came across explaining this. I've also seen some trees which seem to have the sprite clipped inside them as well. 

https://youtube.com/shorts/sh6s17WnWBM?feature=share

**Comment by lenovobenben** [comment]
> I do see the same problem. And it's only on trees that are at a certain distance. Once you go near they stop flickering, and when you move away they start flickering again. Using Vulkan, async shaders disabled, auto cpu accuracy, docked mode, gpu drivers up to date

the same problem. the OpenGL is OK, but it is too slow

**Comment by lenovobenben** [comment]
this is the log. how to send you it?
the both (OpenGL Vulkan) have the same problem. @G-Spawn

**Comment by G-Spawn** [comment]
> this is the log. how to send you it? the both (OpenGL Vulkan) have the same problem. @G-Spawn [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/9389517/yuzu_log.txt)
> 
> the log is here. thx.

It doesnt help either that you are using newer gpu driver. Ideally stick to 512.95 until nvidia stops breaking newer drivers. Pascal and maxwell gpu are hindered mostly by it. This wont help the tree flickering issue but it's still better than busted drivers.

**Comment by HurricanePootis** [comment]
I am having this problem both on OpenGL and Vulkan on my AMD RX 580 using either the RadeonSI Mesa driver, or the ACO RADV driver (AMDVLK crashes yuzu for me).

This problem has been going on for a long time, and happens on both clang and gcc.

**Comment by MichaelW1980** [comment]
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/9457006/yuzu_log.txt)
**Yuzu Version:** 1150
**Graphics Card:** GTX 1660 ti, 6GB
**Driver Version:**  516,94 (DCH)

All settings are default, not counting the controller, which i changed so it uses my Switch Pro Controller, nor counting my audio settings, which i changed to use specific sound playback or recording devices. I have the same issue. All sorts of trees have the same issue for me. Had it for a long time. It doesnt seem to make a difference, wether i change to OpenGL or change any settings to whatever else.

**Comment by HurricanePootis** [comment]
Has there been any progress on bisecting this issue?

**Comment by dml360** [comment]
> Has there been any progress on bisecting this issue?

Unfortunately not yet

**Comment by HurricanePootis** [comment]
Can anyone confirm or deny that this issue persits on the RDNA and RDNA2 cards? Like the 5000 series or the 6000 series with the Mesa drivers on Linux or the window drivers?

**Comment by Geartwo** [comment]
> Can anyone confirm or deny that this issue persits on the RDNA and RDNA2 cards? Like the 5000 series or the 6000 series with the Mesa drivers on Linux or the window drivers?

Yes I have that problem on a RX 6800 XT on Windows with the 22.8.2 Drivers

**Comment by atefhaloui** [comment]
Still happening with latest nvidia driver 522.25 and my 3070 card on both vulkan and OGL no matter config is set.
I can see it as soon as I exit the resurrection shrine and seeing the distant trees.

This does not happen on Cemu 2.0 though (ogl and vulkan).

**Comment by MikeOwaR** [comment]
Is there any news on when this might be fixed?

**Comment by RickSelden** [comment]
Yes please fix this Yuzu team.  It would please a lot of Patreon supporters 👍

**Comment by HurricanePootis** [comment]
Has there been any progress for this?

**Comment by dml360** [comment]
> Houve algum progresso nesse sentido?

No

**Comment by HurricanePootis** [comment]
Has there been any progress for this?

**Comment by MikeOwaR** [comment]
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11120608/yuzu_log.txt)
[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/11120609/yuzu_log.txt.old.txt)

Hi, this is still an issue :( Log files attached

**Comment by savvasdalkitsis** [comment]
This definitely looks like a LOD issue. I get this as well with many trees but weirdly enough i got it with a stone pillar too which make it obvious that somehow the engine keeps flicking between the high and low detail models.

You can see this clearly in this clip:

https://www.youtube.com/watch?v=Dx8BX1pFm8E

**Comment by HurricanePootis** [comment]
Any update on this? 🙃

**Comment by RickSelden** [comment]
I tried this a few days ago and I'm not 100% sure yet but...
It appeared to be fixed in the latest experimental build on Vulkan.  The only reason I'm not certain is because on that session, I updated to the latest experimental, then disabled FPS limit and enabled the 60 FPS mod.

Those 2 settings changes differ from when I get the flickering trees glitch.

So I now need to confirm that its actually fixed in the build and not just a consequence of no-FPS-limit + 60 fps mod.
